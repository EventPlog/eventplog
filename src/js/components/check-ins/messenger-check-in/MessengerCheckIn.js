import React from 'react';
import styled from 'styled-components';

import tmnDefaultBarCode from '../../../../img/tmn_bar_code.png';

const StyleMessengerCheckIn = styled.div`
  > img {
    width: 100%;
  }
`

const MessengerCheckIn = ({ event = {} }) =>
  <StyleMessengerCheckIn>
    <p>Scan this code on Facebook messenger to check into this event</p>
    <img src={event.barcode_url } />
  </StyleMessengerCheckIn>

export default MessengerCheckIn;
