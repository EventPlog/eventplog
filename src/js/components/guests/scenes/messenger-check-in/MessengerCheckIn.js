import React from 'react';
import styled from 'styled-components';

import tmnDefaultBarCode from 'img/tmn_bar_code.png';

const StyleMessengerCheckIn = styled.div`
  > img {
    width: 100%;
  }
`

const MessengerCheckIn = ({ event = {} }) =>
  <StyleMessengerCheckIn>
    <p>Ask guests to scan this code on Facebook messenger to check into this event.</p>
    <img src={event.barcode_url || tmnDefaultBarCode } />
  </StyleMessengerCheckIn>

export default MessengerCheckIn;
