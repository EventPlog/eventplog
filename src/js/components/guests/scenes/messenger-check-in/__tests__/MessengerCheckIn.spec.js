import React from 'react';
import MessengerCheckIn from '../MessengerCheckIn';
import { shallow } from 'enzyme';

describe('MessengerCheckIn', () => {
  it('should render correctly', () => {
    const wrapper = shallow( <MessengerCheckIn  /> );

    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('img').length).toEqual(1);
  })
});