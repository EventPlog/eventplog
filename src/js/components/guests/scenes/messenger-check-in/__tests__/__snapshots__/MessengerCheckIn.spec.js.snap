// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessengerCheckIn should render correctly 1`] = `
<styled.div>
  <p>
    Ask guests to scan this code on Facebook messenger to check into this event.
  </p>
  <img
    src="tmn_bar_code.png"
  />
</styled.div>
`;
